index.html

```diff
@@ -140,6 +140,12 @@ <h4>Optional Events</h4>
       <h3><dfn>Source</dfn> interface</h3>
 
       <pre class="idl">
+      interface SourceOptions {
+        attribute DataFactory? factory;
```

Comment:
I wasn't around for the discussion in #6, but I actually have a slight preference for using `dataFactory` as variable name.

The reason for this is that implementations may accept other kinds of internal _factories_ as well, for example a `nodeFactory`, `numberFactory`, `errorFactory`, ...
For this reason, the `factory` name may be too generic.

Eitherway, it should not be a problem to support for `factory` and `dataFactory` for ensuring backwards-compability.

Reply:
@bergos @RubenVerborgh I think you both use `factory` in N3.js and parser-jsonld, personally I might also have slight preference for `dataFactory` which makes things crystal clear including code like
```js
import dataFactory from '@rdfjs/data-model'

// ....

const parser = new Parser({ dataFactory })
```

Reply:
Preference for `dataFactory` confirmed; will support both in N3.js if adjusted.